composer.json

```diff
@@ -23,9 +23,9 @@
         "sensio/framework-extra-bundle"        : "^3.0",
         "symfony/monolog-bundle"               : "^2.8",
         "symfony/swiftmailer-bundle"           : "^2.3",
-        "symfony/symfony"                      : "^3.2-beta1",
+        "symfony/symfony"                      : "^3.2-rc1",
         "twig/extensions"                      : "^1.3",
-        "twig/twig"                            : "^1.27",
```

Comment:
Do we really need to bump Twig version in composer.json? I mean, do we really have strict dependency with Twig v1.28? it would be enough `composer up twig/twig` - Composer will install 1.28 and freeze it in composer.lock anyway since we have `^` in composer.json.